manager/controllers/default/header.php

```diff
@@ -77,8 +77,17 @@ public function render()
      */
     public function setPlaceholders()
     {
+
+        if($this->modx->getOption('manager_use_fullname') == true) {
+            $userProfile = $this->modx->user->getOne('Profile');
+            $username = $userProfile->get('fullname');
+        } else {
+            $username = '';
```

Comment:
@rtripault no probleme. it's better to make a call than missing a problem :) :+1: